src/manager/TaxiPoolManager.java

```diff
@@ -0,0 +1,11 @@
+package manager;
+
+import entity.Car;
+
+import java.util.List;
+
+public interface TaxiPoolManager {
```

Comment:
I believe this class should be renamed with more appropriate name. Because words Pool and Manager doesn't reflect the actual meaning of your class, it would be great to rename to TaxiParkInformationProvider, but in this case you had to create a TaxiPark interface and his realization.